app/models/queries/filters/shared/user_name_filter.rb

```diff
@@ -70,7 +70,7 @@ def sql_concat_name
       when :firstname_lastname
         "LOWER(CONCAT(users.firstname, CONCAT(' ', users.lastname)))"
```

Comment:
Postgres can receive mutlitple arguments for concat, so we could just rewrite as 
```suggestion
        "LOWER(CONCAT(users.firstname, ' ', users.lastname))"
```